lib/semian/net_http.rb

```diff
@@ -0,0 +1,85 @@
+require 'semian'
+require 'semian/adapter'
+require 'net/https'
+require 'open-uri'
```

Comment:
> Perhaps I'll add code that lets you add more errors?

Sounds good to me.

Otherwise another thing you can do it to register the exception name, not a reference:

```ruby
error.class.name == 'OpenURI::HTTPError'
```

It doesn't work with subclasses, but allow you to handle exceptions that shouldn't be loaded by Semian.